src/utils/NucleicAcidSearchEngine.cpp

```diff
@@ -846,21 +858,27 @@ class NucleicAcidSearchEngine :
     {
       IDFilter::filterQueryMatchesByScore(id_data, fdr_ref, fdr_cutoff);
       OPENMS_LOG_INFO << "Search hits after FDR filtering: "
-               << id_data.getMoleculeQueryMatches().size() << endl;
+                      << id_data.getMoleculeQueryMatches().size()
+                      << "\nIdentified spectra after FDR filtering: "
+                      << id_data.getDataQueries().size() << endl;
     }
   }
 
 
   void generateLFQInput_(IdentificationData& id_data, const String& out_file)
   {
-    // mapping: (oligo, adduct) -> charge -> RTs
-    map<pair<NASequence, String>, map<Int, vector<double>>> rt_info;
+    // mapping: (oligo, adduct) -> charge -> (precursor intensity, RT)
+    map<pair<NASequence, String>,
+        map<Int, vector<pair<double, double>>>> rt_info;
```

Comment:
Hard to read :) typedefs?

Reply:
ehh.. maps within a map? Are you sure?
is not `map<a, map<b, c>>` better represented as map `< (a,b), c>` in terms of efficiency?
depends a bit on the use case I guess (do you want to count charges, for example?)

Reply:
@cbielow: I don't think efficiency is a concern at that point. The code only gets executed once per tool run. In `map<pair<a, b>, c>`, the `a`s are stored multiple times, which may make it more or less efficient.